apps/web/src/app/host-dashboard/page.tsx

```
@@ -0,0 +1,1118 @@
+const handleTogglePropertyStatus = (propertyId: number): void => {
+    setProperties(properties.map((p: Property) => 
+        p.id === propertyId 
+            ? { ...p, status: p.status === 'active' ? 'inactive' : 'active' }
+            : p
+    ));
+};
+
+  const handleAddProperty = (e: { preventDefault: () => void; }) => {
+    e.preventDefault();
+    const price = Number.parseInt(newProperty.price);
+  if (isNaN(price) || price <= 0) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)